### PR TITLE
Only send machine fingerprint when not logged in

### PR DESCRIPTION
**Pull request by davidkelly**
While at it, no longer grabbing it at all when starting up, as that was just a dev thing.  Should consider not sending MAC address too, but we may revisit that later.

Test Plan:

You need 2 clients.  On localhost, you are automatically an admin, so one client on localhost, the other client on a different machine.  When they pop in, look in PAL.  If they are logged in, the PAL will display the machine fingerprint as {000.,}.  When not logged in, the actual fingerprint.  Login and logout during a session, the machine fingerprint should come/go as expected (only when _not_ logged in).  

While at it, look at the PAL on the other client, and verify you don't see the user name or machine fingerprint when you don't have kick rights.

### CODE REVIEW

File: libraries/networking/src/NodeList.cpp [hunk 1]
```diff
@@ -373,9 +373,8 @@ void NodeList::sendDomainServerCheckIn() {
 
             packetStream << hardwareAddress;
 
-            // add in machine fingerprint
-            QUuid machineFingerprint = FingerprintUtils::getMachineFingerprint();
-            packetStream << machineFingerprint;
+            auto accountManager = DependencyManager::get<AccountManager>();
```
Comment (zfox23) [inline libraries/networking/src/NodeList.cpp]: I liked the comment that was here before :)

Comment (davidkelly) [inline libraries/networking/src/NodeList.cpp]: ohh good point.  In fact, I should expand on it...  Just a sec...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7750e86c135185cbd7b637e9de295ccb477a0801 available at: https://gist.github.com/351703d348a8eaa2692ba1700c054dac<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-7750e86.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-7750e86.dmg

**Comment by davidkelly** [comment]
test this please

**Comment by OmegaHeron** [comment]
Will be testing this once built

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4a25b727d88927c1b0294c3dae3dfd97c00734e8 available at: https://gist.github.com/351703d348a8eaa2692ba1700c054dac<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-4a25b72.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-4a25b72.dmg

**Comment by OmegaHeron** [comment]
@davidkelly - could you check to be sure this is in sync with current master?  It's exhibiting a bug that was fixed prior to RC30...  that being scripted agent for audio injector/tutorial can't connect and gets caught in a connect/disco cycle.  I just checked 5822 and it does not exhibit issue.

**Comment by OmegaHeron** [comment]
Looks like #9248 addresses my previous.  Waiting to see.

**Comment by davidkelly** [comment]
apologies -- will update...

**Comment by davidkelly** [comment]
pushing now -- updated to latest master

**Comment by OmegaHeron** [comment]
Sorry @davidkelly - I suspect that won't do it as the disco bug on scripted agent seems to have been reintroduced in #9227 and is being fixed in #9248.  I'll test 9248 then once it's merged it should kill the AC issue for this PR letting it pass.  I need more coffee for this much head spinning.  :)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 50554e934e4c52700e1674573d88b716abf4ee45 available at: https://gist.github.com/351703d348a8eaa2692ba1700c054dac<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-50554e9.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-50554e9.dmg

**Comment by davidkelly** [comment]
K pushed again, after #9248 was merged

**Comment by hifi-gustavo** [comment]
Build console output for commit: 08b6770315e198df279fa3201390bb3b1169504e available at: https://gist.github.com/351703d348a8eaa2692ba1700c054dac<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-08b6770.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9244-08b6770.dmg

**Comment by OmegaHeron** [comment]
Testing Windows 10

If I'm getting the logic correct this passes.  :)  Check my screen caps and state descriptions below.
General smoke test = pass.  I also checked various other combinations of primary/secondary logged in/!logged in -- display name set/!set and in each case PAL (eventually showed what seems to be correct values).

Myself on Windows 10 connected to sandbox on same machine
My second self on second machine (Linux) connecting to Win10 sandbox via LAN

Both primary and secondary AV logged in.  Primary AV (Omega) has display name set, secondary does not.

hifi-snap-by-omegaheron-on-2016-12-20_21-01-35

Both primary and secondary AV logged in.  Primary AV (Omega) and secondary (Heez) has display name.

hifi-snap-by-omegaheron-on-2016-12-20_21-04-11

Primary continues to be logged in, secondary logged out (with display name unset)
In this case I logged out secondary and you can see fingerprint is shown + display name.

hifi-snap-by-omegaheron-on-2016-12-20_21-06-06

Secondary exits domain and returns (not logged in, display name unset).  I suppose key issue is - AV logged in id string = 0...0, !logged in it = something clearly not 0. 

hifi-snap-by-omegaheron-on-2016-12-20_21-06-12

**Comment by davidkelly** [comment]
That looks right to me!

**Pull request merged**